vendor/github.com/mattn/go-colorable/noncolorable.go

```diff
@@ -0,0 +1,61 @@
+package colorable
+
+import (
+	"bytes"
+	"io"
+)
+
+// NonColorable hold writer but remove escape sequence.
+type NonColorable struct {
+	out     io.Writer
+	lastbuf bytes.Buffer
+}
+
+// NewNonColorable return new instance of Writer which remove escape sequence from Writer.
+func NewNonColorable(w io.Writer) io.Writer {
+	return &NonColorable{out: w}
+}
+
+// Write write data on console
+func (w *NonColorable) Write(data []byte) (n int, err error) {
```

Comment:
__[gometalinter]__
* [gocyclo] cyclomatic complexity 13 of function (*NonColorable).Write() is high (> 10)

<!-- Comment by [SideCI](https://sideci.com) -->